libs/utils/utils.js

```diff
@@ -75,25 +120,54 @@ export function loadStyle(href, callback) {
   return link;
 }
 
+/**
+ * Load template (page structure and styles).
+ */
+export async function loadTemplate() {
+  const template = getMetadata('template');
+  if (!template) return;
+  const name = template.toLowerCase().replace(/[^0-9a-z]/gi, '-');
+  document.body.classList.add(name);
+  const { miloLibs, projectRoot } = getConfig();
+  const base = miloLibs && MILO_TEMPLATES.includes(name) ? miloLibs : projectRoot;
+  const styleLoaded = new Promise((resolve) => {
+    loadStyle(`${base}/templates/${name}/${name}.css`, resolve);
+  });
+  const scriptLoaded = new Promise((resolve) => {
+    (async () => {
+      try {
+        await import(`${base}/templates/${name}/${name}.js`);
+      } catch (err) {
+        // eslint-disable-next-line no-console
+        console.log(`failed to load module for ${name}`, err);
+      }
+      resolve();
+    })();
+  });
+  await Promise.all([styleLoaded, scriptLoaded]);
+}
+
 export async function loadBlock(block) {
   const { status } = block.dataset;
   if (!status === 'loaded') return block;
```

Comment:
Can this ever be true?  `!status` will coerce the string into a bool which can not equal the `'loaded'` string.

Also I don't see where `block.dataset.status` is assigned `loaded`

Reply:
> Also I don't see where block.dataset.status is assigned loaded

Good point. I think this came from boilerplate and we can remove.